sympy/parsing/c/c_parser.py

```diff
@@ -234,16 +236,43 @@ def transform_var_decl(self, node):
                 while child.kind == cin.CursorKind.NAMESPACE_REF:
                     child = next(children)
 
-                while child.kind == cin.CursorKind.TYPE_REF:
-                    child = next(children)
+                # while child.kind == cin.CursorKind.TYPE_REF:
+                #     child = next(children)
```

Comment:
any reason for commenting this out?

Reply:
I'd suggest to leave it as it is. We don't need type references for primary data types anyway.

Reply:
Alright... Actually, I was trying to parse the types from `stdint.h` such as `int8_t`, `int16_t` which are exactly similar to `int8`, `int16` of `numpy`. I left that as comment accidentally.